src/lib/__tests__/date.test.ts

```diff
@@ -1,59 +1,365 @@
 import { isNull } from "lodash"
-import moment, { Moment } from "moment"
-import { exhibitionPeriod, exhibitionStatus } from "lib/date"
+import moment, { Moment } from "moment-timezone"
+import {
+  dateRange,
+  exhibitionStatus,
+  dateTimeRange,
+  singleDate,
+  singleDateTime,
+  singleTime,
+  formattedOpeningHours,
+  datesAreSameDay,
+  timeRange,
+} from "lib/date"
 
-describe("date", () => {
-  describe("exhibitionPeriod", () => {
-    it("includes the start and end date", () => {
-      const period = exhibitionPeriod(
-        moment("2011-01-01T00:00-0400"),
-        moment("2014-04-19T00:00-0400")
+describe("date formatting", () => {
+  describe(timeRange, () => {
+    it("if within same am/pm only display one am or pm", () => {
```

Comment:
I think you may not be including the “it” when you’re thinking of these descriptions. Here the description should probably read:

```suggestion
    it("only displays one am or pm if within same am/pm", () => {
```

…as in, “it only displays one am or pm if within same am/pm”.

Reply:
Ah! Yes I was never considering `it` part of the sentence.

Reply:
Also, the ‘it’ refers to your subject specified with the `describe` function.